tests/Frameworks/CodeIgniter/Version_2_2/ddshim.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+/* CodeIgniter expects the CWD to be set to the root directory but the builtin
+ * CLI SAPI server will not do this:
+ * https://www.php.net/manual/en/features.commandline.differences.php */
+chdir(__DIR__);
+
+require 'index.php';
```

Comment:
Do we need this shim anymore now that we bootstrap in the bridge? :)

Reply:
Yes -- that `chdir` is needed because CodeIgniter expects the current working directory to be the one containing the index, but that's not how the CLI web SAPI sets it up, unlike other SAPIs.